taskchampion/src/replica.rs

```diff
@@ -392,4 +416,34 @@ mod tests {
         let uuid = Uuid::new_v4();
         assert_eq!(rep.get_task(uuid).unwrap(), None);
     }
+
+    #[test]
+    fn expire() {
```

Comment:
Would it be worth adding a second test case to detail what happens with conflicts in this case - e.g replica 1 deletes task, a year later replica 2 marks task is done, another year later the two replicas are synced? (I assume it essentially creates a new, maybe incomplete, task with the modifications?)

Reply:
I believe (but can check) that is tested in detail by the sync tests, but it makes sense as an integration test.

Reply:
47b0a8903325b64df99738de83b0b630ef543a92

Reply:
Ah good point!